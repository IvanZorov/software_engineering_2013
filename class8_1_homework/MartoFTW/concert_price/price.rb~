require 'csv'

all = []
CSV.foreach(ARGV[0]) do |row|
        if (row[3].to_i == ARGV[1].to_i)
                all << row
        end 
end
all = all.sort_by{|a,b,c,d,e,f| a}
CSV.open("price_result.csv","w") do |csv|
        all.each do |row|
                csv << row
	end
end


