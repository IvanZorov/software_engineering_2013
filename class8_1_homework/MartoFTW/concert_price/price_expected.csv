Jaya,Reggae,Bulgaria,16,Pork Pie,27/01/2012
Bon Jovi,Rock,Bulgaria,16,Vasil Levski,10/10/2010
